src/routers/movie.js

```diff
@@ -0,0 +1,21 @@
+const express = require("express");
+const {
+    getAllMovies,
+    createMovie,
+    getMovieById,
+    updateMovie
+} = require('../controllers/movie');
+
+const router = express.Router();
+
+// In index.js, we told express that the /customer route should use this router file
+// The below /register route extends that, so the end result will be a URL
+// that looks like http://localhost:4000/customer/register
```

Comment:
These comments are also incorrect as this file deals with the movie router not customer

Reply:
I think this was defaulted in the code, I didn't write this haha